<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QSvgWidget" name="svg_CaTSper" native="true">
         <property name="minimumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>44</width>
           <height>44</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_CaTSper">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Semibold</family>
           <pointsize>28</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>CaTSper</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_import">
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>101</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>8</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Import THz Files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_filenames">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>24</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_deploy">
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>101</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>8</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Deploy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_clear">
         <property name="minimumSize">
          <size>
           <width>101</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>101</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>8</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Clear Memory</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSvgWidget</class>
   <extends>QWidget</extends>
   <header>PyQt6.QtSvgWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
